src/components/ByCategory/ByCategoryData.js

```diff
@@ -54,17 +53,14 @@ function ByCategoryData({ kids }) {
 
   useEffect(() => {
```

Comment:
You can just remove the whole useEffect block as it is no longer doing anything.